wework/src/i18n/locales/en/common.json

```
@@ -238,8 +248,18 @@
+    "project_automation_waiting_approval": "Awaiting approval",
+    "project_automation_starting": "Starting",
+    "project_automation_waiting_runtime": "Awaiting runtime",
     "project_automation_running": "Running",
+    "project_automation_cancelling": "Cancelling",
+    "project_automation_succeeded": "Succeeded",
+    "project_automation_failed": "Failed",
+    "project_automation_cancelled": "Cancelled",
+    "project_automation_skipped": "Skipped",
+    "project_automation_unknown": "Status unknown",
     "project_automation_completed": "Completed",
+    "project_automation_run_details": "View run details",
```

Biome (2.5.6)

[error] The key project_automation_run_details was already declared.

(lint/suspicious/noDuplicateObjectKeys)